test/bpf/nat-test.c

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/* Copyright (C) 2021 Authors of Cilium */
+
+// source file for nat_test.h.
+// It contains contains main functions to run test functions nat_test.h.
+// It is used to perform unit test on functions in nat.h.
+
+#include "tests/nat_test.h"
+
+void setUp(void) {
+    // set stuff up here
+}
+
+void tearDown(void) {
+    // clean stuff up here
+}
+
+
+int main(int argc, char *argv[])
+{
+    test_snat_v4_track_local();
+    return 0;
```

Comment:
There was another test case previously. Any reason for the removal?

Reply:
There was a test on function `snat_v4_new_mapping` which uses only mock helper functions. It was written only as a demo and not complete. So I removed it and keep only `test_snat_v4_track_local` which is a comprehensive test.